### PR TITLE
Broken sidebar bug fix 

### LINKED ISSUE #1172: 🐛 Bug: Blog content List in Sidebar is broken
### Describe the bug

Related to https://github.com/json-schema-org/website/issues/1142
The content list on the left side is distorted and shows a bullet at the end of list item.

### Steps To Reproduce

1. Go to [JSON Schema joins the OpenJS Foundation and 2022 updates](https://json-schema.org/blog/posts/json-schema-joins-the-openjsf#json-schema-is-hiring!-(at-postman)) blog
2. Observe the left sidebar in desktop view.

### Expected Behavior

For this blog to be consistent with other blogs, the bullet points should not show up at the end of every list item and it should be visible clearly on the screen without getting cut off.

### Screenshots

image

### Device Information [optional]

```markdown
- OS: MacOS
- Browser: Chrome
- version: 131.0.6778.140
```

### Are you working on this issue?

Yes

### Do you think this work might require an [Architectural Decision Record (ADR)]? (significant or noteworthy)

No

### PR DESCRIPTION
**Pull request by Utkarsh-123github**
**What kind of change does this PR introduce?**

**Issue Number:**

-  Closes #1172 

**Screenshots/videos:**
Screenshot 2024-12-17 114731

**If relevant, did you update the documentation?**
No

**Summary**

This PR basically resolves the issue of broken content list in sidebar which is mentioned in the issue : https://github.com/json-schema-org/website/issues/1172

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

File: pages/learn/index.page.tsx [hunk 1]
```diff
@@ -39,7 +37,7 @@ export default function Welcome() {
           link='https://tour.json-schema.org/'
         />
       </div>
-      <DocsHelp markdownFile={markdownFile} />
+      <DocsHelp />
```
Comment (benjagm) [inline pages/learn/index.page.tsx:40]: Why this change?

**Review by benjagm — CHANGES_REQUESTED** [review]
The main change looks great but I left a comment regarding another unrelated change you added.

File: pages/learn/index.page.tsx (hunk 1 above)
Comment (Utkarsh-123github) [reply pages/learn/index.page.tsx:40]: Oh, I think it is from one of the previous PR, where there was a redirect to a wrong page. I will correct this asap and will update the PR.

Comment (Utkarsh-123github) [reply pages/learn/index.page.tsx]: Actually , this was part of some other PR , that solves redirecting issue , and by mistake it got reflected in this PR too.
I have rectified the changes. Kindly review the PR and suggest some changes if applicable.

**Review by benjagm — APPROVED** [review]
Great job!! Thanks!

**Comment by Utkarsh-123github** [comment]
Thanks @benjagm , I'm looking forward to make more meaningful contributions.
